### PR TITLE
SITL VTOL: scale outputs based on motor count (param VT_MOT_COUNT)

### PR DESCRIPTION
**Pull request by jlecoeur**
**Describe problem solved by the proposed pull request**
In the SITL MAVLink module, the outputs are scaled based on the number of motors determined from the vehicle type. For tiltrotors and other types of VTOLs, the vehicle type is not sufficient to determine the number of motors.
Example: Tricopter tiltrotor with servo on 4th output. The servo output is scaled in range [0,1] (as a motor) instead of [-1, 1].
 
**Describe your preferred solution**
The parameter VT_MOT_COUNT is used to determine the number of motors and correctly scale outputs in the ranges [0,1] or [-1,1].

**Describe possible alternatives**
Getting rid of the dedicated simulation MAVLink module?

### CODE REVIEW

**Comment by jlecoeur** [comment]
I am not sure what the CI failure is. 
In both commits the standard_vtol model goes down shortly after take off. It is of type `MAV_TYPE_VTOL_RESERVED2` which is affected by the first commit, however the second commit should have fixed it but obviously did not. Any input?

File: ROMFS/px4fmu_common/init.d-posix/1040_standard_vtol [hunk 1]
```diff
@@ -34,7 +34,7 @@ then
 
 	param set VT_F_TRANS_DUR 5
 	param set VT_F_TRANS_THR 0.75
-	param set VT_MOT_COUNT 4
+	param set VT_MOT_COUNT 5
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d-posix/1040_standard_vtol]: I believe this is the number of MC motors, not total.

Comment (jlecoeur) [reply ROMFS/px4fmu_common/init.d-posix/1040_standard_vtol]: fair enough

**Comment by jlecoeur** [comment]
CI error in SITL tests, the problem is described in #11380

**Pull request closed without merging**